src/modules/apiComments.js

```diff
@@ -0,0 +1,34 @@
+const saveComment = async (data) => {
+  const apiComment = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/RfJpwLbuNZkYmafYdHPm/comments/';
+
+  try {
+    const response = await fetch(apiComment, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+
+    if (response.ok) {
+      console.log('Comentario enviado exitosamente');
+    }
+  } catch (error) {
+    console.log('Error en la solicitud:', error);
+  }
+};
+
+const fetchShowComments = async (showId) => {
+  try {
+    const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/RfJpwLbuNZkYmafYdHPm/comments?item_id=${showId}`);
```

Comment:
- [ ] Since you've already declared a variable for the API earlier, kindly call the variable to avoid code repetition. 👍